charts/atlantis/values.yaml

```diff
@@ -451,6 +451,40 @@ initContainers: []
 #   image: alpine:latest
 #   command: ['sh', '-c', 'echo The init container is running! && sleep 10']
 
+# Install providers/plugins into a path shared with the Atlantis pod
+initConfig:
+  enabled: false
+  image: alpine:latest
+  imagePullPolicy: IfNotPresent
+  # sharedDir is set as env var INIT_SHARED_DIR
+  sharedDir: /plugins
+  workDir: /tmp
+  sizeLimit: 100Mi
+  script: |
+    #!/bin/sh
+    set -eoux pipefail
+
+    # Terragrunt
+    TG_VERSION="v0.47.0"
+    TG_SHA256_SUM="98d45f6bfbfae84b51364c1ad6920f09ecb4d834908b0535e4e331a9fc6fc75b"
+    TG_FILE="${INIT_SHARED_DIR}/terragrunt"
+    wget https://github.com/gruntwork-io/terragrunt/releases/download/${TG_VERSION}/terragrunt_linux_amd64 -O "${TG_FILE}"
+    echo "${TG_SHA256_SUM}  ${TG_FILE}" | sha256sum -c
+    chmod 755 "${TG_FILE}"
+    terragrunt -v
+
+    # terragrunt-atlantis-config
+    TAC_VERSION="1.16.0" # without v
```

Comment:
so for Terragrunt users this will need to be updated when needed making the atlantis helm-chart dependant on Terragrunt releases, why this need to be hardcoded?

Reply:
This is more a suggestion and showing what you can do since the default values are `initConfig.enabled.false` and you'll have to change the default values to enable. I don't think it's correct to use latest, otherwise your infra might change every time the atlantis pod restarts.

What are your suggestions about this? I can make it generic so it doesn't include a version.

Reply:
that is fine, maybe mention this is an example script because I bet people will think it is only for terragrunt.
an I guess this could be used for provider cache in TF ( aws provider I mean)

Reply:
Ok, I'll update the description above to say it's an example.